internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/controller/kafka/consumer/AdminConsumptionTaskIntegrationTest.java

```diff
@@ -243,6 +272,73 @@ public void testUpdateAdminOperationVersion() {
     }
   }
 
+  @Test(timeOut = 2 * TIMEOUT)
+  public void testAdminConsumptionTaskWithSpecificWriterId() throws IOException {
+    int adminConsumptionMaxWorkerPoolSize = 3;
+    // Use a specific version to test the serialization and deserialization of admin operation.
+    int writerSchemaId = AdminOperationSerializer.LATEST_SCHEMA_ID_FOR_ADMIN_OPERATION - 10;
+
+    Properties parentControllerProps = new Properties();
+    parentControllerProps.put(ADMIN_CONSUMPTION_MAX_WORKER_THREAD_POOL_SIZE, adminConsumptionMaxWorkerPoolSize);
+    parentControllerProps.put(ADMIN_CONSUMPTION_CYCLE_TIMEOUT_MS, 3000);
+
+    try (
+        VeniceTwoLayerMultiRegionMultiClusterWrapper venice =
+            ServiceFactory.getVeniceTwoLayerMultiRegionMultiClusterWrapper(
+                new VeniceMultiRegionClusterCreateOptions.Builder().numberOfRegions(1)
+                    .numberOfClusters(1)
+                    .numberOfParentControllers(1)
+                    .numberOfChildControllers(1)
+                    .numberOfServers(1)
+                    .numberOfRouters(1)
+                    .replicationFactor(1)
+                    .parentControllerProperties(parentControllerProps)
+                    .build());
+        ControllerClient parentControllerClient = new ControllerClient(
+            venice.getClusterNames()[0],
+            venice.getParentControllers().get(0).getControllerUrl())) {
```

Comment:
Could you please make it a shared cluster for this test file?

Reply:
Updated, bring the init into setup()